test_unstructured_ingest/unit/pipeline/reformat/test_chunking.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import annotations
+
+import os
+import tempfile
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+
+from test_unstructured.unit_utils import (
+    FixtureRequest,
+    Mock,
+    example_doc_path,
+    function_mock,
+    method_mock,
+)
+from unstructured.documents.elements import CompositeElement
+from unstructured.ingest.interfaces import ChunkingConfig, PartitionConfig
+from unstructured.ingest.pipeline.interfaces import PipelineContext
+from unstructured.ingest.pipeline.reformat.chunking import Chunker
+
+ELEMENTS_JSON_FILE = example_doc_path(
+    "test_evaluate_files/unstructured_output/Bank Good Credit Loan.pptx.json"
+)
+
+
+class DescribeChunker:
+    """Unit tests for ingest.pipeline.reformat.chunking.Chunker"""
+
+    # -- Chunker.run() -----------------------------------------------------------------------------
+
+    # -- integration test --
+    def it_creates_json(self, _ingest_docs_map_: Mock):
+        chunking_config = ChunkingConfig(chunking_strategy="by_title")
+        pipeline_context = PipelineContext()
+        partition_config = PartitionConfig()
```

Comment:
These are all short and can be inlined in the `Chunker` call immediately below. In general, the only reason you want to introduce a local variable for something like this is if you need access to the object later in the test.

Besides compactness, placing them inline avoids implying to the reader that these need to be accessed later. Like the time I spent discovering they aren't used anywhere else would be unnecessary.